index.js

```diff
@@ -4,17 +4,19 @@ const priceMinFilter = document.querySelector('#input-search-price-min');
 const priceMaxFilter = document.querySelector('#input-search-price-max');
 const bookAuthorFilter = document.querySelector('#input-search-author');
 const yearFilter = document.querySelector('#input-search-year');
-const searchBtn = document.querySelector('#btn-search');
-const tableBody = document.querySelector('#table-similar-books tbody');
-const tableth = document.querySelector('table th');
+const similarTableBody = document.querySelector('#table-similar-books tbody');
+const allBooksTableBody = document.querySelector('#table-all-books tbody');
 const similarBookContainer = document.querySelector('.similar-book-container');
 const defaultImage = './product-not-found.png';
 const similarBookCount = document.querySelector('#similar-book-count');
+// const upArrowSort = document.querySelector('.fa-arrow-up-short-wide');
+// const downArrowSort = document.querySelector('.fa-arrow-down-short-wide');
```

Comment:
remove the commented code. if not needed

Reply:
Done